src/github.com/stellar/gateway/bridge/handlers/request_handler_builder.go

```diff
@@ -42,10 +42,20 @@ func (rh *RequestHandler) Builder(w http.ResponseWriter, r *http.Request) {
 
 	sequenceNumber, err := strconv.ParseUint(request.SequenceNumber, 10, 64)
 	if err != nil {
-		errorResponse := protocols.NewInvalidParameterError("sequence_number", request.SequenceNumber, "Sequence number must be a number")
-		log.WithFields(errorResponse.LogData).Error(errorResponse.Error())
-		server.Write(w, errorResponse)
-		return
+		accountResponse, err := rh.Horizon.LoadAccount(request.Source)
```

Comment:
I think we shouldn't do this here. This block will be executed when `request.SequenceNumber` is empty but also when the value is not a valid uint64. This may be confusing for users. We should explicitly check if `request.SequenceNumber` is empty.